src/System-Support-Tests/SystemDictionaryTest.class.st

```diff
@@ -102,7 +102,7 @@ SystemDictionaryTest >> testSetOrganizationSetBackPointer [
 
 	| sd so |
 	sd := SystemDictionary new.
-	so := SystemOrganizer new.
+	so := RPackageOrganizer new.
```

Comment:
so `ro`, no?

Reply:
I gave better names (systemDictionary and packageOrganizer because I **hate** abbreviations in code)